src/components/MerchantsPage/gam/MegaGamProgressBar.tsx

```diff
@@ -0,0 +1,107 @@
+import moment from 'moment';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import ProgressBar from './ProgressBar';
+
+const PROGRESS_BAR_COLOR = '#CF6E8A';
+
+const centsToDollars = (cents: number) =>
```

Comment:
nit: we should make a util that does this for us across all of the frontend since this is a common thing

Reply:
i can do this in a followup